src/game/base.rs

```diff
@@ -1460,4 +1568,9 @@ impl PostFlopGame {
     pub fn get_state(&self) -> &State {
         &self.state
     }
+
+    #[inline]
+    pub fn is_partially_solved(&self) -> bool {
```

Comment:
We should document that this is a new function in the changelog/v0.1.1 release notes